synapse/push/httppusher.py

```diff
@@ -271,17 +264,12 @@ async def _unsafe_process(self):
                     )
                     self.backoff_delay = HttpPusher.INITIAL_BACKOFF_SEC
                     self.last_stream_ordering = push_action["stream_ordering"]
-                    pusher_still_exists = await self.store.update_pusher_last_stream_ordering(
+                    await self.store.update_pusher_last_stream_ordering(
                         self.app_id,
                         self.pushkey,
                         self.user_id,
                         self.last_stream_ordering,
                     )
-                    if not pusher_still_exists:
-                        # The pusher has been deleted while we were processing, so
-                        # lets just stop and return.
-                        self.on_stop()
-                        return
```

Comment:
Why has this been removed?

Reply:
[`update_pusher_last_stream_ordering`](https://github.com/matrix-org/synapse/blob/02e588856ae26865cd407dd6302aa3deecffe198/synapse/storage/databases/main/pusher.py#L183-L191) always returns `None`.

Reply:
Ah right. I wonder if that was a mistake

Reply:
I'm not sure since it calls [`simple_update_one`](https://github.com/matrix-org/synapse/blob/02e588856ae26865cd407dd6302aa3deecffe198/synapse/storage/database.py#L1601) which also doesn't return anything. I suspect this was just copied and pasted from above.